manifests/params.pp

```diff
@@ -8,6 +8,7 @@
   $port = 8000
   $root_dir = '/opt/graphite'
   $version = '0.9.12'
+  $worker_processes = 2
```

Comment:
Should we change this to 4 here, already?!

Reply:
Well, once it's configurable, we can hopefully put the 4 in the govuk_puppet hieradata. I haven't thought that far ahead though.